docs/developers/reference/api/linea-estimategas.mdx

```diff
@@ -0,0 +1,98 @@
+---
+description: Reference content for the linea_estimateGas method.
+image: /img/socialCards/reference-content-for-the-lineaestimategas-method.jpg
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# `linea_estimateGas`
+
+:::info
+
+`linea_estimateGas` is being reactivated on testnet from May 28. Mainnet activation is scheduled
+for July 24, 2024. 
+
+It will only be fully compatible with endpoints using the Besu client — if it is unavailable, try 
```

Comment:
Will this be available for users running a Besu follower node? (our existing run a node instructiions) If not, we may want to clarify.

Reply:
Yes, my understanding is that this is the case